### DISCUSSION TITLE
Solcon TV boxes suddenly show "No internet connection possible"

### DISCUSSION THREAD

**Discussion by b-brefeld** [fabianishere/udm-iptv#363] [Q&A]
Since, this morning all my Solcon TV boxes suddenly decided to stop working.
I remember it still showing some info but no visuals when my IPTV wasn't set up properly.
But this time, I only get an error message "No internet connection possible" (both when connected to the "iptv" vlan and the default lan that doesn't cast iptv) I would have assumed it would at least get an internet connection with 1 of the 2.

I tried reconfiguring the script and restarting the UDM Pro. I have no idea what might be causing these issues as I don't see any errors in my output either (I think).

Here are my logs:
```=== Configuration ===
WAN Interface: eth8
WAN VLAN: 188 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 10.0.0.0/8  10.252.0.0/16  10.253.0.0/16  217.166.0.0/16
LAN Interfaces: br188
IGMP Proxy quickleave disabled: true
IGMP Proxy debug: true
=== IP Link and Route ===
47: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.71.156.217/20 scope global iptv
       valid_lft forever preferred_lft forever
10.0.0.0/8 via 10.71.144.1 metric 247 
10.71.144.0/20 proto kernel scope link src 10.71.156.217 
=== Service Logs ===
Dec 30 16:47:25 Brefeld-UDMPRO udm-iptvd[56224]: DEBUG[mcast_recv_igmp@748]: src addr = 192.168.1.11
Dec 30 16:47:25 Brefeld-UDMPRO udm-iptvd[56224]: received interface  15
Dec 30 16:47:25 Brefeld-UDMPRO udm-iptvd[56224]: WARNING[mcast_recv_igmp@753]: Don't exist this VIF 15
Dec 30 16:47:25 Brefeld-UDMPRO udm-iptvd[56224]: DEBUG[main@653]: Data arrive
Dec 30 16:47:25 Brefeld-UDMPRO udm-iptvd[56224]: DEBUG[mcast_recv_igmp@748]: src addr = 192.168.1.11
Dec 30 16:47:25 Brefeld-UDMPRO udm-iptvd[56224]: received interface  15
Dec 30 16:47:25 Brefeld-UDMPRO udm-iptvd[56224]: WARNING[mcast_recv_igmp@753]: Don't exist this VIF 15
Dec 30 16:47:25 Brefeld-UDMPRO udm-iptvd[56224]: DEBUG[main@653]: Data arrive
Dec 30 16:47:25 Brefeld-UDMPRO udm-iptvd[56224]: DEBUG[mcast_recv_igmp@748]: src addr = 192.168.1.11
Dec 30 16:47:25 Brefeld-UDMPRO udm-iptvd[56224]: received interface  15
```

As for the configuration I use:
Device: UDM PRO,
Unifi-OS: v4.0.21,
Network: 8.6.9,
Latest version of the current script,
Provider: Solcon,
I'm using the preconfigured Solcon option, and it should be fine as it worked before.

It always worked fine and suddenly stopped. Any help would be greatly appreciated.

**Comment by dmutsaers** [comment]
Solcon is my provider too, but I don't have the same issues with my Arris VIP 5202.
UXG Lite (4.0.20)
Network 8.6.9
I assume the "Don't exist this VIF 15" error has something to do with your issue.
Sorry I can't be of more help.

**Reply by b-brefeld** [reply]
Would you be willing to share your related network configuration and potentially your udm-iptv config file?

I might be able to spot something

**Comment by dmutsaers** [comment]
![image](https://github.com/user-attachments/assets/eaa7d24f-b603-488c-8493-5318cc16d4a0)
<img width="1360" alt="Snipaste_2025-01-08_10-51-05" src="https://github.com/user-attachments/assets/8657e7be-137a-4bad-b366-4e9a1713f6a3" />
![image](https://github.com/user-attachments/assets/5cf4991d-5cff-4f9e-a1f9-94afcc036070)
![image](https://github.com/user-attachments/assets/e4306fed-51f0-4044-a39e-5468ffc4e5c5)
[udm-iptv.txt](https://github.com/user-attachments/files/18306027/udm-iptv.txt)

**Comment by Sandwich3399** [comment]
Hi, did you manage to solve this issue? I have exactly the same issues with Solcon's Delta Fiber network (CAIW-EAS network, VLAN 188 for IPTV). It worked perfectly from nov-2023 until 20th december 2024, now we keep getting the message that no internet is available (Internetverbinding niet mogelijk) on the VIP5202 Arris IPTV boxes. The original Amino IPTV box does work..

**Reply by b-brefeld** [reply]
I haven't been back at the related address yet.
So sadly, I haven't been able to test this out.

If I figure something out, I will make sure to post it here.
Please do the same if you find anything useful.

I notice we both have issues with Arris boxes. This might be related.
Can you try to restart the Amino box and see if it still works properly? (Just to make sure it is limited to Arris boxes at your end?)

**Comment by Sandwich3399** [comment]
Yes, I can restart the Amino box without problems, only the Arris VIP 5202 boxes don't work. Solcon helpdesk is looking into it as well, they've acknowledged that they see the Amino box connecting without any issues. 

Do you only use Arris boxes, or do you also have an Amino box? And if you have the Amino box as well, does it work while the Arris boxes don't?

**Reply by b-brefeld** [reply]
Sadly, I only have Arris boxes.
This is why I'm intrigued to hear that you only have the issue with Arris.
This opens up the possibility that it's related to Arris boxes rather than the overall settings.

**Comment by dmutsaers** [comment]
Could this be related to your issue, or can it be a similar issue?

https://github.com/fabianishere/udm-iptv/pull/368
